doc/runtimes/runtimeDebianFlavor.md

```diff
@@ -0,0 +1,57 @@
+# Oryx runtime base images debian flavor
```

Comment:
Looks like we have some duplicate documentation info, since we have https://github.com/microsoft/Oryx/blob/main/doc/supportedRuntimeVersions.md

Perhaps we want to update the existing one to break it down by debian flavor like this does?

Reply:
Updated